services/streammanager/kafka/kafkamanager.go

```diff
@@ -307,7 +326,7 @@ func NewProducer(destination *backendconfig.DestinationT, o common.Opts) (*Produ
 			codecs[avroSchema.SchemaId] = newCodec
 		}
 	}
-
+	// TODO: once control-plane changes to get this config from it is in production. We can safely remove this.
```

Comment:
Can you please create a task on Notion to follow up on this?

Reply:
https://www.notion.so/rudderstacks/kafka-over-ssh-via-env-cleanup-after-control-plane-changes-are-merged-b1d1d074073d4d398aab083f6d153ee8?pvs=4